pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,119 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  hwloc,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+  stability = "release";
```

Comment:
Suggest inlining `stability` and `os` since they're only used in the download URL.